code/package.json

```diff
@@ -57,6 +57,7 @@
     "changelog": "pr-log --sloppy --cherry-pick",
     "changelog:next": "pr-log --sloppy --since-prerelease",
     "check": "NODE_ENV=production node ../scripts/check-package.js",
+    "ci": "yarn task --task check --no-link --start-from=install && yarn lint && yarn test",
```

Comment:
What's this for and why's it called "ci"?

Reply:
Ah yeah, ehm, in my frustration getting the build green, I added this command. Basically mimicking some of the steps from the circle ci checks.

I can remove it, but maybe this is generally useful? I could also add the scrip units test to it. Then basically this covers all static checks and unit-tests of the CI.

Reply:
called ci-tests now